.azure-pipelines/unit_test.yml

```diff
@@ -0,0 +1,34 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+trigger:
+  - main
```

Comment:
Just to confirm. There is no PR trigger since the pipeline is optional at the moment?

Reply:
Yeah, after this PR merge, I will open PR trigger from ADO side.